packages/web/src/components/TextField/_TextField.scss

```diff
@@ -46,16 +46,11 @@
     width: 100%;
 }
 
-.TextField:hover > .TextField__input,
-.TextField__input:hover {
-    border-color: theme.$input-border-color-hover;
-}
-
-.TextField:focus-within,
-.TextField:focus-within:hover,
-.TextField > .TextField__input:focus {
+.TextField:focus-within .TextField__input,
```

Comment:
This line makes focus shadow visible even when `focus-visible` is not fired on other elements. 🤔 

Do we need it? I'd very like to see the same behavior on all components.

Reply:
We probably don't need it. I will remove it.